core/src/main/java/cz/xtf/core/config/TestCaseContext.java

```diff
@@ -0,0 +1,47 @@
+package cz.xtf.core.config;
+
+import static cz.xtf.core.config.OpenShiftConfig.OPENSHIFT_NAMESPACE;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestCaseContext {
```

Comment:
My feeling is this new class should have better defined responsibilities. 

As Is it is very dumb. Just holder of 2 fields. Can we expose some API which will better define what is this class for?

Reply:
The best would be to have JUnit5 injecting `TestInfo` but it's not able to do so.

Reply:
I see. So I start to understand purpose of this class. It is to hold information about TestCase.

Could that be documented. 

In that case I wonder do we need `Map<String, String> testcaseToNamespaceMap`? Do we need to keep history of all namespaces?

Isn't it enough to keep just one field `private static String testCaseNamespace`?

Reply:
I've added `testcaseToNamespaceMap` as preparation for parallel run execution. I would like to keep it.

Reply:
Can it be set as resolved?

Reply:
Have you considered moving class to `junit5/src/main/java/cz/xtf/junit5/`.

Reply:
Problem is that it would create dependency from core -> junit5 module. I would like to avoid it and better have the opposite. Let other modules depend on core.

Reply:
Makes totally sense. Resolving.